common.blocks/i-bem/__dom/i-bem__dom.js

```diff
@@ -764,7 +764,7 @@ DOM = BEM.decl('i-bem__dom',/** @lends BEMDOM.prototype */{
      * @private
      * @param {String} name Nested element name
      * @param {String} [modName] Modifier name
-     * @param {String} [modVal] Modifier value
+     * @param {String|Boolean} [modVal=true] Modifier value
```

Comment:
Are you sure that this change should be here? What about `elem()` method's JSDoc at [#L786](https://github.com/bem/bem-core/pull/903/files#diff-fe894527c21cbc977d795eac412ce183R786)?